src/main/java/data/DatabaseController.java

```diff
@@ -0,0 +1,1032 @@
+package data;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import pathfinding.Node;
+import pathfinding.ConcreteNode;
+
+import static java.sql.DriverManager.println;
+
+public class DatabaseController {
```

Comment:
Brace placement - don't worry about it too much, but the rest of the project (...I think? I'll check at the meeting today) is using braces on the line after classes & methods, e.g.

    public class DatabaseController
    {
        //Code
    }

This should be as simple as changing the relevant code styling options in intellij and then telling it to reformat the code.